e2e/testinfra/terraform/prow/service_accounts.tf

```diff
@@ -23,11 +23,17 @@ resource "google_project_iam_member" "test-runner-iam" {
     "roles/container.admin",
     "roles/gkehub.admin",
     "roles/monitoring.viewer",
-    "roles/secretmanager.secretAccessor",
+    "roles/secretmanager.admin",
```

Comment:
Do we need the `secretmanager.admin` role? The only missing role is `secretmanager.viewer`. We can have both `secretmanager.secretAccessor` and `secretmanager.viewer`.

Reply:
The code destroys secrets when they're invalid and creates them when they don't exist. So it probably needs admin.

Reply:
With https://github.com/GoogleContainerTools/kpt-config-sync/pull/1077 the SA needs a broader range of permissions. It needs permission now to add/disable secret versions when the key expires, and potentially create a new secret if running on a new environment that needs bootstrapping. We could remove the ability to bootstrap the initial secret if we want to keep the permissions more minimal, although that means it needs to be done manually when creating a new test environment.

Reply:
ACK